Assets/Scripts/PowerUps/VelocityModifier.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class VelocityModifier : MonoBehaviour
```

Comment:
This script looks like it either requires `PlayerMovement` to function or `PlayerMovement` requires it to function. Consider adding the `[RequireComponent(typeOf()))]` tag in one of the classes.

Reply:
Okay sures!